test/v2/client/test_client.go

```diff
@@ -397,7 +433,7 @@ func (c *TestClient) GetIndexedChainState() core.IndexedChainState {
 }
 
 // GetRetrievalClient returns the test client's retrieval client.
-func (c *TestClient) GetRetrievalClient() clients.RetrievalClient {
+func (c *TestClient) GetRetrievalClient() validator.ValidatorClient {
```

Comment:
nit: rename method to `GetValidatorClient`

Reply:
fixed